nuxt.config.js

```diff
@@ -132,5 +135,11 @@ export default {
     */
     extend (config, ctx) {
     }
+  }, 
+  publicRuntimeConfig: {
+    spacesId: process.env.CONTENTFUL_SPACES_ID,
+    contentfulAccessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
+    contentfulFullUrl: process.env.CONTENTFUL_FULL_URL,
```

Comment:
I wouldn't normally comment on a variable name, but contentfulFull is a mouthfulfull. Can you make this `contentfulUrl` or `contentfulApiUrl` or something like that?

You know how some words when you say/see them over and over and they start to make no sense? I think that just happened with me and `contentfulFullUrl` 😆